### ISSUE TITLE
Unable to import example data on site build

### ISSUE THREAD

**Issue by ThatGerber** [bradp/vv#59]
Version: vv version 1.5.0

Ran through wizard to create a new site (`vv create`). While creating the new site, it fails to import the dummy data if the WordPress importer is not installed:

````
==> default: Installing WordPress (release version) in example/htdocs...
==> default: Downloading WordPress 4.1 (en_US)...
==> default: Using cached file '/home/vagrant/.wp-cli/cache/core/en_US-4.1.tar.gz'...
==> default: Success: WordPress downloaded.
==> default: Success: Generated wp-config.php file.
==> default: Success: WordPress installed successfully.
==> default: Error: WordPress Importer needs to be installed. Try 'wp plugin install wordpress-importer --activate'.
==> default: /srv/www/example
==> default: Setting up blueprint default...
==> default: Blueprint set up.
````

**Comment by leogopal** [comment]
Have you found a solution? 
I am running into the same issue.

**Comment by bradp** [comment]
Didn't realize that WP-CLI needs the WP importer to exist in order to import sample data.

Not sure how I didn't catch this in testing. I'm not sure of the best solution, as we don't want to add this plugin if a blueprint is used, or if wp-content is from a git repo.

Any ideas?

**Comment by bradp** [comment]
Fixed in a1fc71ebe0eb, will be in the next released version.
